assets/app/models/Document.js

```diff
@@ -2,27 +2,68 @@ var _ = require('underscore');
 var Backbone = require('backbone');
 var yaml = require('yamljs');
 
+var fileTypes = {
+  'md': ['md', 'markdown'],
+  'yml': ['yml', 'yaml']
+};
+
+var yamlDefaults = [
```

Comment:
I think so, because if there is a layout that doesn't have a corresponding file Jekyll will break the build and it will be hard for folks without that context to know how to fix it